modules/persistent-storage-hostpath-pod.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-hostpath.adoc
+
+[id="persistent-storage-hostpath-pod_{context}"]
+= Mounting the hostPath share in a privileged pod
+
+After the persistent volume claim has been created, it can be used inside by an application. The following example demonstrates mounting this share inside of a pod.
```

Comment:
Using a PV + PVC with HostPath volume works, but it's not very useful - basically only when there is no other persistent storage available to the cluster, then HostPath PVs are the last resort.

It's more common to use HostPath volumes in-line directly in a Pod, for example when the pod wants to configure something on the nodes and it already knows the path that it wants to access there:

```
apiVersion: v1
kind: Pod
metadata:
  name: pod-name
spec:
  containers:
    ...
    securityContext:
      privileged: true
    volumeMounts:
    - mountPath: /host/etc/motd.conf <1>
      name: hostpath-privileged
  ...
  volumes:
    - name: hostpath-privileged
      hostPath:
          path: /etc/motd.conf <2>
<1> The path to mount the hostPath share inside the privileged pod.
<2> The path on the host to share into the privileged pod.
```
As result, the pod can see host's /etc/motd.conf inside the pod as /host/etc/motd.conf and can configure "message of the day" service on the host, without accessing the host directly (the example is purely artificial).

I am not sure where to squeeze this information - it's not really a *persistent* storage.